### DISCUSSION TITLE
warden db import

### DISCUSSION THREAD

**Discussion by atty31** [wardenenv/warden#815] [Help & Support]
### Version of Warden

in-dev

### Operating System and Installation Method

Ubuntu, Linux

### Describe the Bug

Hello,

I am trying to do a database import: **cat data.sql | warden db import** , but i'm stuck with one table with a lot of varchar columns.
Getting this error: 
`Row size too large (> 8126). Changing some columns to TEXT or BLOB may help. In current row format, BLOB prefix of 0 bytes is stored inline`.

Running this query: 
`SHOW VARIABLES LIKE 'innodb_strict_mode';`  will show that the value is set to `ON`.  Following the instructions from: https://stackoverflow.com/questions/22637733/mysql-error-code-1118-row-size-too-large-8126-changing-some-columns-to-te i want to turn it off with `SET GLOBAL innodb_strict_mode = OFF;` . But I'm getting: 
`42000][1227] Access denied; you need (at least one of) the SUPER privilege(s) for this operation`

How can I run that query with root access maybe ?

Thank you

**Comment by navarr** [comment]
If you're using the Magento template, the root credentials for the db are `root:magento`
